### PR TITLE
[Docs] Spark engine support for steps: MapValues, Imputer, OneHotEncoder, DropFeatures; DateExtractor (partial)

### PR DESCRIPTION
**Pull request by jillnogold**
ML-2709

### CODE REVIEW

File: docs/serving/available-steps.md [hunk 1]
```diff
@@ -38,10 +38,11 @@ Click on the step names in the following sections to see the full usage.
...
+| [mlrun.feature_store.DateExtractor](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DateExtractor)  | Extract a date-time component. Spark engine supports extracting the time parts from the date data.|
+| [mlrun.feature_store.DropFeatures](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DropFeatures) | Drop features from feature list. Supported by the Spark engine. |
+| [mlrun.feature_store.Imputer](../api/mlrun.feature_store.html#mlrun.feature_store.steps.Imputer) | Replace None values with default values. Supported by the Spark engine. |
+| [mlrun.feature_store.MapValues](../api/mlrun.feature_store.html#mlrun.feature_store.steps.MapValues) | Map column values to new values. Supported by the Spark engine. |
+| [mlrun.feature_store.OneHotEncoder](../api/mlrun.feature_store.html#mlrun.feature_store.steps.OneHotEncoder) | Create new binary fields, one per category (one hot encoded). Supported by the Spark engine. | 
 | [mlrun.feature_store.SetEventMetadata](../api/mlrun.feature_store.html#mlrun.feature_store.steps.SetEventMetadata) | Set the event metadata (id, key, timestamp) from the event body. |
```
Comment (davesh0812) [inline docs/serving/available-steps.md:46]: All the mlrun.feature_store steps not include SetEventMetadata are supported by storey, pandas and spark engines, SetEventMetadata supported only by storey engine.

Also in the Other section we Have FeaturesetValidator that supported only with storey and pandas engines.

**Review by davesh0812 — COMMENTED** [review]
Thanks! just a minor comment

File: docs/serving/available-steps.md [hunk 2]
```diff
@@ -38,11 +38,12 @@ Click on the step names in the following sections to see the full usage.
...
+| [mlrun.feature_store.DropFeatures](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DropFeatures) | Drop features from feature list. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.Imputer](../api/mlrun.feature_store.html#mlrun.feature_store.steps.Imputer) | Replace None values with default values. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.MapValues](../api/mlrun.feature_store.html#mlrun.feature_store.steps.MapValues) | Map column values to new values. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.OneHotEncoder](../api/mlrun.feature_store.html#mlrun.feature_store.steps.OneHotEncoder) | Create new binary fields, one per category (one hot encoded). Supported also by the Pandas and Spark engines. | 
+| [mlrun.feature_store.SetEventMetadata](../api/mlrun.feature_store.html#mlrun.feature_store.steps.SetEventMetadata) | Set the event metadata (id, key, timestamp) from the event body. Supported also by the Pandas and Spark engines.|
```
Comment (davesh0812) [inline docs/serving/available-steps.md:46]: SetEventMetadata supported only by storey (my bad)

File: docs/serving/available-steps.md [hunk 3]
```diff
@@ -101,5 +102,5 @@ Click on the step names in the following sections to see the full usage.
 ## Other
 | Class name                                       | Description                                   |   
 |--------------------------------------------------|-----------------------------------------------------------|
-| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validate feature values according to the feature set validation policy. | 
+| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validate feature values according to the feature set validation policy. Supported also by the Pandas and Spark engines. |
```
Comment (davesh0812) [inline docs/serving/available-steps.md:105]: FeaturesetValidator supported only by storey and pandas! not supports by Spark.

**Review by davesh0812 — COMMENTED** [review]
:)

**Comment by jillnogold** [comment]
replaced by https://github.com/mlrun/mlrun/pull/3035

**Pull request closed without merging**